src/main/java/website/frontrow/board/Bubble.java

```diff
@@ -34,7 +34,6 @@
     
     private static final long TIME_FLOAT_UPWARDS = 50;
 
-    // Time to kill the bubble is dependent on the time it has float upwards before.
     @SuppressWarnings("checkstyle:magicnumber")
```

Comment:
Again if the checkstyle is fixed, remove the suprression, if it isn't keep the explanation.

<layout